pkg/phlaredb/phlaredb.go

```diff
@@ -49,6 +50,7 @@ type Config struct {
 
 	MinFreeDisk                uint64        `yaml:"min_free_disk_gb"`
 	MinDiskAvailablePercentage float64       `yaml:"min_disk_available_percentage"`
+	BlockExpiry                time.Duration `yaml:"block_expiry"`
```

Comment:
Ah, just have noticed that our retention policy settings are placed at root. These are especially "vague":

```yaml
pyroscopedb:
  enforcement_interval:
  disable_enforcement:
```

I don't think that there is a nice way to add `retention_policy` prefix and not break compatibility, just something we should keep in mind.

Reply:
Yeah, this is a good point. Perhaps it's worth looking into a way to move these keys off the root, but still check for it on the root for legacy purposes.